routes/index.js

```diff
@@ -6,7 +6,8 @@ const appVersion = require(path.join(__dirname, '..', 'package.json')).version;
 /* GET home page. */
 router.get('/', function (req, res) {
   res.render('index', {
-  	version : appVersion
+  	version : appVersion,
+  	env : process.env.NODE_ENV
```

Comment:
What do you think to renaming the property to `isProduction` and the value being the expression `process.env.NODE_ENV === 'production'`. This way we won't need the custom helper for handlebars and can use the built-in `if` helper.